dotcom-rendering/src/model/newsletterIllustrations.ts

```diff
@@ -0,0 +1,101 @@
+export type NewsletterIlustrations = Partial<Record<string, string>>;
```

Comment:
```suggestion
export type NewsletterIllustrations = Partial<Record<string, string>>;
```

Reply:
Also, not sure why we need [Partial](https://www.typescriptlang.org/docs/handbook/utility-types.html#partialtype) here?

Reply:
I think in some typescript configs, using `Record<string,string>` makes the compiler assume the value for any arbitrary key will be a string, rather than string or undefined - but  that doesn't apply here so you're right, we don't need it.
Looking again, we don't actually need to define the `NewsletterIllustrations` type at all, we can just type the object as a being a `Record<string,string>`